content/en/apps/examples/covid-education.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "COVID-19 Education and Training for CHWs"
+linkTitle: "COVID-19 Education"
+weight: 
+description: >
+ Education and training for Community Health Workers to combat COVID-19.
```

Comment:
Use "address" or "mitigate" instead of "combat", and remove period from descriptions. I couldn't find reference to that in the doc style guide (yet!), but couldn't find another description with a period.

```suggestion
 Education and training for Community Health Workers to address COVID-19
```